docs/pages/machine-id/guides/jenkins.mdx

```diff
@@ -0,0 +1,170 @@
+---
+title: Using Teleport Machine ID with Jenkins
+description: Using Teleport Machine ID with Jenkins
+---
+
+Jenkins is an open source automation server that is frequently used to build
+Continuous Integration and Continuous Delivery (CI/CD) pipelines.
+
+In this guide, we will demonstrate how to migrate existing Jenkins pipelines to
+utilize Machine ID with minimal changes.
+
+## Prerequisites
+
+You will need the following tools to use Teleport with Jenkins.
+
+- The Teleport Auth Service and Proxy Service version >= 9.0.0, deployed on
+  your own infrastructure or managed via Teleport Cloud.
+- `ssh` OpenSSH tool
+- Jenkins
+
+## Architecture
+
+Before we begin, it should be noted that Jenkins is a tool that is notoriously
+difficult to secure. Machine ID is one part of securing your infrastructure,
+but it alone is not sufficient. Below we will provide some basic guidance which
+can help improve the security posture of your Jenkins install.
+
+**Single-host deployments:** The simplest Jenkins deployments have the
+controller (process that stores configuration, plugins, UI) and agents (process
+that executes tasks) run on the same host. This deployment model is simple to
+get started with, however any compromise of the `jenkins` user within a single
+pipeline can lead to the compromise of your entire CI/CD infrastructure.
+
+**Multihost deployments:** A slightly more complex, but more secure deployment
+is running your Jenkins controllers and agents on different hosts and pinning
+workloads to specific agents. This is a improvement over the simple deployment
+because you can limit the blast radius of the compromise of a single pipeline
+to a subset of your CI/CD infrastructure instead of all of your infrastructure.
+
+We strongly encourage the use of the second deployment model whenever possible,
+with ephemeral hosts and IAM joining when possible. When using Machine ID with
+this model, create and run Machine ID bots per-host and pin particular
+pipelines to a worker. This will allow you to give each pipeline the minimal
+scope for server access, reduce the blast radius if one pipeline is
+compromised, and allow you to remotely audit and lock pipelines if you detect
+malicious behavior.
+
+<Figure align="left" bordered caption="Jenkins Deployments">
+  ![Jenkins Deployments](../../../img/machine-id/machine-id-jenkins.png)
```

Comment:
Is there more diagram design work that needs to be done here?

Reply:
I removed to diagram for now. @ardeche07 is working with design to get us a new diagram.